### PR TITLE
added spotify clone (WIP)

### PR DESCRIPTION
**Pull request by DYG307**

### CODE REVIEW

File: lyriks_starter_code/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Setup
+
+To set up a development environment, please follow these steps:
+
+1. Clone the repo
+
+   ```shell
+    git clone https://github.com/JavaScript-Mastery-PRO/project1_team4_repository.git
+   ```
```
Comment (AlexLuo602) [inline lyriks_starter_code/README.md:55]: this sucks pls fix

**Comment by AlexLuo602** [comment]
hi

File: lyriks_starter_code/README.md (hunk 1 above)
Comment (AlexLuo602) [reply lyriks_starter_code/README.md:55]: hi

**Review by AlexLuo602 — APPROVED** [review]

File: lyriks_starter_code/README.md (hunk 1 above)
Comment (DYG307) [reply lyriks_starter_code/README.md:55]: no u

**Pull request merged**